opentelemetry-instrumentation/tests/test_semconv.py

```diff
@@ -0,0 +1,222 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from unittest import TestCase
+from unittest.mock import Mock, patch
+
+from opentelemetry.instrumentation._semconv import (
+    OTEL_SEMCONV_STABILITY_OPT_IN,
+    _OpenTelemetrySemanticConventionStability,
+    _OpenTelemetryStabilitySignalType,
+    _set_status,
+    _StabilityMode,
+)
+from opentelemetry.trace.status import StatusCode
+
+
+def semconv_stability_mode(mode):
+    def decorator(test_case):
+        @patch.dict(os.environ, {OTEL_SEMCONV_STABILITY_OPT_IN: mode})
+        def wrapper(*args, **kwargs):
+            _OpenTelemetrySemanticConventionStability._initialized = False
+            _OpenTelemetrySemanticConventionStability._initialize()
+            return test_case(*args, **kwargs)
+
+        return wrapper
+
+    return decorator
+
+
+class TestOpenTelemetrySemConvStability(TestCase):
+    @semconv_stability_mode("")
+    def test_default_mode(self):
+        self.assertEqual(
+            _OpenTelemetrySemanticConventionStability._get_opentelemetry_stability_opt_in_mode(
+                _OpenTelemetryStabilitySignalType.HTTP
+            ),
+            _StabilityMode.DEFAULT,
+        )
```

Comment:
It will also be `_StabilityMode.DEFAULT` for `_OpenTelemetryStabilitySignalType.DATABASE`, is that correct?

Reply:
That's correct. For any kind of signal type, we have only one default, which stands for "" or the absence of the environment variable.